src/permission/PermissibleBase.php

```diff
@@ -137,13 +155,10 @@ public function recalculatePermissions() : void{
 	}
 
 	public function clearPermissions() : void{
-		$permManager = PermissionManager::getInstance();
-		$permManager->unsubscribeFromAllPermissions($this->getRootPermissible());
-
-		$permManager->unsubscribeFromDefaultPerms(false, $this->getRootPermissible());
-		$permManager->unsubscribeFromDefaultPerms(true, $this->getRootPermissible());
+		PermissionManager::getInstance()->unsubscribeFromAllPermissions($this->getRootPermissible());
 
 		$this->permissions = [];
+		$this->rootPermissions = [];
```

Comment:
Need to re-review this. I'm not sure it makes sense, but it's here to break cyclic dependencies between `PermissibleBase` and `PermissionAttachmentInfo`.